doc/api/http.md

```diff
@@ -117,6 +117,10 @@ http.get({
 <!-- YAML
 added: v0.3.4
 changes:
+  - version:
+      - REPLACEME
```

Comment:
nit: since it's semver major, it will never be backported, no need for a list here.
```suggestion
  - version: REPLACEME
```